serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,37 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
```

Comment:
lol yeah, honestly a concern for another team as well as another day I think, since I didn't create the Service Mesh variables :slightly_smiling_face: @JStickler could you raise this with your team please? We use `{ServerlessProductName}` so maybe ya'll could update this to something similar for your product?

Reply:
Yeah, we could probably change that to cause less confusion.  I'll open a JIRA.

Reply:
Thanks @JStickler!